src/formatDate.js

```diff
@@ -50,7 +50,58 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  let newDate = '';
+  let index = 0;
+  const dateArr = date.split(fromFormat[3]);
+
+  for (let i = 0; i < 3; i++) {
+    if (fromFormat.includes(toFormat[i])) {
+      index = fromFormat.indexOf(toFormat[i]);
+    } else if (fromFormat.includes('YYYY')) {
+      index = fromFormat.indexOf('YYYY');
+    } else {
+      index = fromFormat.indexOf('YY');
+    }
+
```

Comment:
I'm confused what is this index for... It might be better to name the variable `indexFrom` or `indexTo`. but it's up to you))